src/net/smooth/parser.js

```diff
@@ -193,6 +203,7 @@ function createSmoothStreamingParser(parserOptions={}) {
         ? parse(q.getAttribute(key))
         : parse[q.getAttribute(key)];
     }
+    obj["codec"] = extractCodec(q.getAttribute("FourCC") || "", obj);
```

Comment:
- the attribute is `codecs` (plural)
- this attribute should only exist for audio or video adaptations